### ISSUE TITLE
Comment are not always syntax-colored correctly

### ISSUE THREAD

**Issue by lukaszcz** [anoma/vscode-juvix#22]
See the attached image. The first three dashes (-) in the third comment line are black and non-italic.

![image](https://user-images.githubusercontent.com/62751/209225902-7cba0eb0-43ed-4119-ac33-c9419503b2ba.png)

**Comment by jonaprieto** [comment]
Yes, this is indeed a bug but in the Juvix compiler, not in the extension. We commented on this some days ago.

**Comment by jonaprieto** [comment]
I confirm at v0.1.29 this is no longer an issue:

![image](https://github.com/anoma/vscode-juvix/assets/1428088/e70ee62d-c87d-4b88-9402-08ba3dbae8ad)
